model-optimizer/mo/graph/graph.py

```diff
@@ -747,7 +746,8 @@ def check_shapes_consistency(self):
             if not data_node.has('shape'):
                 data_nodes_with_wrong_shapes.append((data_node.name, "no shape attribute"))
                 continue
-            if data_node.shape is not None and not isinstance(data_node.shape, np.ndarray):
+            if data_node.shape is not None and not isinstance(data_node.shape, np.ndarray) and \
+                    dynamic_dimension_value not in data_node.shape:
```

Comment:
why is `dynamic_dimension_value` not allowed for shape consistency?

Reply:
This is a very strange condition. How come I have added it?! Removed.